pyproject.toml

```diff
@@ -67,6 +67,7 @@ dev = [
     "pytest-mypy",
     "pytest-remotedata",
     "sphinx",
+    "sphinx-gallery",
```

Comment:
Do we need this?

Reply:
Yes, it is **required**... Although `sphinx-gallery` achieves same result, after working with it, I see the following disadvantages:

* It requires a lot of time to build docs because of figure output captures.
* A big refactor is required to switch. Customization and maintenance are more difficult also.

Reply:
Sorry, perhaps I'm confused - don't we need `nbsphinx-gallery` here instead of `sphinx-gallery`?

Reply:
No, `nbsphinx-gallery` does not exist, it's just `nbsphinx`. But that's already declared, so I don't understand why do we need `sphinx-gallery` if we decided not to use it 😇

Reply:
As claimed in [official nbsphinx docs](https://nbsphinx.readthedocs.io/en/0.7.0/subdir/gallery.html):

> nbsphinx does not provide any gallery styles, but you can easily use the styles from Sphinx-Gallery by installing it. However, you can also create your own CSS styles if you prefer (then you don’t need to install Sphinx-Gallery). You can load your CSS files with html_css_files.

I am not sure how to build the gallery by making use of custom CSS but that option would lead to not depend on sphinx-gallery :thinking:

Reply:
Oh, I see! Now I understand, all good then.